### PR TITLE
Feature/at 8634 ready to submit

### PR DESCRIPTION
**Pull request by DGabriel-CCPO**
When the user selects DICTO as their contracting office, it will trigger submission flow to submit the acquistion package to DITCO.

### CODE REVIEW

File: src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue [hunk 1]
```diff
@@ -1,26 +1,125 @@
...
+          </ol>
+        </div>
+      </div>
+      <div class="ml-10">
+        <v-card
+          class="
+            border1
+            border-base-lighter
+            pa-4"
+          :elevation="2"
```
Comment (ericyoungquist-ccpo) [inline src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue:41]: On `v-card` tag:
1. remove `elevation`
2. add classes `_shadow` and `border-rounded-more`
3. change `pa-4` to `pa-6`

In file `utils.scss` add `!important` to the `._shadow` class

File: src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue [hunk 2]
```diff
@@ -1,26 +1,125 @@
...
+        >
+          <h3 class="mb-2">Your completed package includes:</h3>
```
Comment (ericyoungquist-ccpo) [inline src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue:43]: change `mb-2` to `mb-3` and add class `nowrap`

File: src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue [hunk 3]
```diff
@@ -1,26 +1,125 @@
...
+            <li
+              v-for="(item,idx) in documentList"
+              :key="idx"
+              class="text-base py-1"
+            >
+              {{item}}
+            </li>
+          </ul>
+          <v-btn
+            class="secondary _text-decoration-none px-6 mt-3"
```
Comment (ericyoungquist-ccpo) [inline src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue:54]: change `mt-3` to `mt-6`

File: src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue [hunk 4]
```diff
@@ -1,26 +1,125 @@
...
+            large
+            role="button"
+            :href="downloadPackageLink"
```
Comment (ericyoungquist-ccpo) [inline src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue:57]: sonar cloud is complaining about security hot spot on this - look into using vue sanitize on this link perhaps? we also have a plugin in `plugins/sanitize.ts`

File: src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue [hunk 5]
```diff
@@ -1,26 +1,125 @@
...
+          class="
+            border1
+            border-base-lighter
+            pa-4"
+          :elevation="2"
+        >
+          <h3 class="mb-2">Your completed package includes:</h3>
+          <ul>
+            <li
+              v-for="(item,idx) in documentList"
```
Comment (ericyoungquist-ccpo) [inline src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue:46]: if J&A and MRR, remove `(Template)` from string

File: src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue [hunk 6]
```diff
@@ -1,26 +1,125 @@
...
+            :value.sync="certified"
+            :items="checkboxItem"
+            :boldText="true"
+            :card="false"
+          />
+          <ol class="ml-6">
+            <li class="pb-3">
+              All information in my package is accurate and complete to the best of my knowledge.
+            </li>
+            <li class="pb-3">
```
Comment (ericyoungquist-ccpo) [inline src/steps/11-GeneratePackageDocuments/ReadyToSubmit.vue:29]: remove class on last `li`

File: src/store/acquisitionPackage/index.ts [hunk 7]
```diff
@@ -1450,6 +1450,15 @@ export class AcquisitionPackageStore extends VuexModule {
     this.doSetPackageId(value);
   }
 
+  @Action({rawError: true})
+  public async setDownloadPackageLink(): Promise<string> {
+    const domain = document.location.origin.indexOf("localhost") > 0
```
Comment (ericyoungquist-ccpo) [inline src/store/acquisitionPackage/index.ts:1455]: change to `> -1`

**Review by ericyoungquist-ccpo — CHANGES_REQUESTED** [review]

**Review by ajaynalamala-ccpo — APPROVED** [review]

**Review by ericyoungquist-ccpo — APPROVED** [review]

**Pull request merged**